deploy/docker-compose/README

```diff
@@ -0,0 +1,153 @@
+storescu -v +r +sd -aec ORTHANC localhost 1114 /path/to/my/dicom/*
+```
+
+Navigate to the home page and click *All studies* to confirm data's been uploaded.
+
+### Sample Workflows
+
+Under the **sample-workflows* directory, a couple sample workflow definitions are provided:
+
+- `hello-world.json `: Hello World!
+- `spleen-seg.json`: AI Spleen Segmentation MAP
+- `liver-seg.json`: AI Liver Segmentation MAP
+
+
+#### Hello World
+
+##### Description
+
+This example uses `alpine` image to simply print all files found in the input directory.
+
+1. Deploy the workflow definition to MONAI Deploy Workflow Manager:
+   ```
+   $ curl --request POST --header 'Content-Type: application/json'  --data "@sample-workflows/hello-world.json"  http://localhost:5001/workflows
+
+   {"workflow_id":"849d4683-006b-410c-aa17-d0474ee26b7b"}
+   ```
+   If the `curl` command runs successfully, expect a `workflow_id` to be returned and printed to the terminal.
+1. Navigate to Orthanc, select any study and then click *Send to DICOM Modality* from the menu on the left.
+   In the popup dialog, select **MONAI-DEPLOY** to start a C-STORE request to the Informatics Gateway.
+1. To see the output from the container, run the following commands:
+   ```bash
+   $ docker container list -a | grep alpine
+   # locate the container ID and run the following
+   $ docker logs {CONTAINER ID}
+   # expect a list of files to be printed
+   /var/monai/input/1.2.826.0.1.3680043.2.1125.1.19616861412188316212577695277886020/1.2.826.0.1.3680043.2.1125.1.34918616334750294149839565085991567/1.2.826.0.1.3680043.2.1125.1.60545822758941849948931508930806372.dcm.json
+   /var/monai/input/1.2.826.0.1.3680043.2.1125.1.19616861412188316212577695277886020/1.2.826.0.1.3680043.2.1125.1.34918616334750294149839565085991567/1.2.826.0.1.3680043.2.1125.1.60545822758941849948931508930806372.dcm
+   ...
+   ```
+
+#### AI Spleen Segmentation MAP
+
+##### Description
+
+In this section, we will run the [Spleen Segmentation MAP](https://github.com/Project-MONAI/monai-deploy-app-sdk/tree/main/examples/apps/ai_spleen_seg_app) from the
+[MONAI Deploy App SDK](https://github.com/Project-MONAI/monai-deploy-app-sdk).
+
+1. Download the data set from [here](https://drive.google.com/file/d/1IGXUgZ7NQCwsix57cdSgr-iYErevqETO/view?usp=sharing)
+1. Deploy the workflow definition to MONAI Deploy Workflow Manager:
+   ```
+   $ curl --request POST --header 'Content-Type: application/json'  --data "@sample-workflows/spleen-seg.json"  http://localhost:5001/workflows
+
+   {"workflow_id":"811620da-381f-4daa-854d-600948e67228"}
+   ```
+   If the `curl` command runs successfully, expect a `workflow_id` to be returned and printed to the terminal.
+1. Navigate to Orthanc, select any study and then click *Send to DICOM Modality* from the menu on the left.
+   In the popup dialog, select **MONAI-DEPLOY** to start a C-STORE request to the Informatics Gateway.
+1. Reload the Orthanc study page and expect a new series to be added.
```

Comment:
"Refresh" the study in Orthanc, but this can ONLY be done once the workflow has been completed.